echopop/computation/spatial.py

```diff
@@ -103,10 +103,8 @@ def calculate_transect_distance( dataframe ,
     )
     
 def transform_geometry( dataframe: pd.DataFrame ,
```

Comment:
Here the docstring is missing `range_out`.

After reading through the code though, I feel `range_out` should not be an input argument. I feel the code would be more readable if you always return a 3-element list, but in the range_out=False case, simply ignore the output. Like in `standardize_coordinates`, do:
```python
mesh_trans_df, _, _ = transform_geometry( ... )
```
And the other case the code does not need to change.

This will cut down the test case also.